eng/Packages.Data.props

```diff
@@ -57,6 +57,7 @@
     <PackageReference Update="Microsoft.ServiceFabric.Services" Version="3.3.624" />
     <PackageReference Update="Microsoft.ServiceFabric" Version="6.4.624" />
     <PackageReference Update="Microsoft.SourceLink.GitHub" Version="1.0.0-beta2-18618-05" />
+    <PackageReference Update="Microsoft.Spatial" Version="7.5.3" />
```

Comment:
If I understand correctly, all dependencies will be centralized here from now on. Is the expectation that service teams are responsible for bumping the dependency version numbers in this file?

Reply:
The version will be centralized here to help align versions across the different sdk libraries. Service teams can update this but they should ping us to ensure that the version bump is ok and doesn't break things for others.

Reply:
So if I'm not actively updating this and nobody else is using it, does anybody update it? If somebody else is using it and updates it, is my only chance to find out to watch all the PRs in the repo?

Reply:
Yes I think you have that correct. I suspect what most people will do is check their dependencies of the packages they are using before they release to ensure the package dependencies are understood and if there is some reason it was updated and you guys cannot take that update for some reason then you can always override the version in your project with "VersionOveride" metadata on your PackageReference. However the entire point of this mono repo is to try and enforce consistent dependencies so we should only vary in exceptional cases.

Reply:
Makes sense. This is unlikely to be a problem in practice for us, at least for this particular dependency. I just want to make sure there is a process in place to detect and fix breakage quickly, since mono-repos don't have the benefit of isolation.